cachito/web/models.py

```diff
@@ -47,6 +47,33 @@ class Request(db.Model):
     def __repr__(self):
         return '<Request {0!r}>'.format(self.id)
 
+    @property
+    def state(self):
+        """
+        Get the name of the latest state.
+
+        :return: the name of the latest state or None if one doesn't exist.
+        :rtype: str/None
+        """
+        request_state = self.latest_state
+        if request_state:
+            return RequestStateMapping(request_state.state).name
+
+    @property
+    def latest_state(self):
+        """
+        Get the latest RequestState associated with this request.
+
+        :return: the latest RequestState associated with this request
+        :rtype: RequestState
+        """
+        return (
+            RequestState.query
```

Comment:
Shouldn't this be accessible through the backreference states_history?

Reply:
Yes, but calling that would retrieve all the states associated with the request. I figured it'd be more efficient to grab just the one RequestState. Regardless, after refactoring this PR, this is no longer needed so I'll just remove it.